api/src/lib/crud.js

```diff
@@ -0,0 +1,128 @@
+import koa from 'koa';
+import coBody from 'co-body';
+import koaRoute from 'koa-route';
+
+export default (queries, restricted, excludedRouteMethod) => {
+    const app = koa();
+    let q;
```

Comment:
The lib is called `queries`, which represent a simple list of SQL queries.
I renamed `queries` to `queriesFactory` and `q` to `queries`.
Let me know if you're ok with that !